app/src/main/java/com/sdpteam/connectout/client_validation/ProfileValidationUtils.java

```diff
@@ -0,0 +1,63 @@
+package com.sdpteam.connectout.client_validation;
+
+import android.util.Patterns;
+import android.widget.EditText;
+import android.widget.RadioButton;
+
+public class ProfileValidationUtils {
+
```

Comment:
Same here, ProfileValidator seems more suited

Reply:
does EditProfileValidator works for you ?

Reply:
Sure, it is still clear and straight to the point